include/posix/sys/select.h

```diff
@@ -22,8 +22,15 @@ static inline int select(int nfds, fd_set *readfds,
 			 fd_set *writefds, fd_set *exceptfds,
 			 struct timeval *timeout)
 {
+#ifdef CONFIG_NET_SOCKETS_POSIX_NAMES
```

Comment:
I don't think this is needed. posix headers are only valid when CONFIG_POSIX_API=y, but CONFIG_POSIX_API and CONFIG_NET_SOCKETS_POSIX_NAMES cannot be set simultaneously:
```
config NET_SOCKETS_POSIX_NAMES
	bool "Standard POSIX names for Sockets API"
	depends on !POSIX_API
```